tests/test_hypothesis.py

```diff
@@ -0,0 +1,81 @@
+import random
+
+from hypothesis import given, settings, strategies as st
+import unittest
+from parsigs.parse_sig_api import SigParser, StructuredSig
+
+
+class TestParseSigApi(unittest.TestCase):
+    parser = SigParser()
+
+    @given(st.integers(min_value=1, max_value=30))
+    @settings(max_examples=30)
+    def test_parse_sig_strength(self, integer):
+        sig = f"Take 1 tablet of ibuprofen {integer}mg 3 times every day for 10 weeks"
```

Comment:
rename this file to test_propery_parsig_api (the `test_` starter is for `pytest` to run this script when testing the project)